tests/postgresql/needs.json

```diff
@@ -4,6 +4,7 @@
 			"repository": "git://git.postgresql.org/git/postgresql.git",
 			"commit": "0ed1e572b644e064e480e14d2f6afe03d11638a7",
 			"project": {
+				"max-jobs": 1,
```

Comment:
i would prefer different terminology like "concurrency" over "jobs". "jobs" seems more ambiguous to me. e.g. this could be interpreted to mean "execute one job, then quit" rather than "execute all jobs, one at a time". i also think "concurrency" is better seo